acceptance/tests/invalid_ssl_config.rb

```diff
@@ -36,7 +36,8 @@
     create_remote_file(agent, pxp_agent_config_file(agent), pxp_config_json_using_puppet_certs(master, agent).to_s)
     on agent, puppet('resource service pxp-agent ensure=running')
 
-    assert(is_associated?(master, "pcp://#{agent}/agent"),
+    inventory_retries = 60
+    assert(is_associated?(master, "pcp://#{agent}/agent", inventory_retries),
```

Comment:
The retries here occur when the test receives an inventory response from the broker and that inventory doesn't contain the expected _pcp://#{agent}/agent_ identity.

See my comment below re. a workaround (and a concern with it)